openlibrary/plugins/upstream/code.py

```diff
@@ -48,6 +48,14 @@ class change_photo(change_cover):
 
 del delegate.modes['change_cover']     # delete change_cover mode added by openlibrary plugin
 
+
+class components_test(delegate.page):
+    path = "/_dev/components"
+
+    def GET(self):
+        return render_component('HelloWorld') + render_component('HelloWorld')
```

Comment:
strange that this generic endpoint is mapped to a single random HelloWorld component, but yolo

Reply:
Renamed to `components/HelloWorld`